src/Calc.php

```diff
@@ -22,8 +22,24 @@ public function sayHello(DateTimeImmutable $date): void
         echo 'Hello, ' . $date->format('j. n. Y');
     }
 
+    public $displayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyaaaayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyaaa;
+
+    public function sayaHello(DateTimeImutable $date): void
+    {
+        echo 'Hello, ' . $date->format('j. n. Y');
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Call to method format() on an unknown class Acme\DateTimeImutable.